go.mod

```diff
@@ -1,17 +1,16 @@
 module github.com/openshift/aws-ebs-csi-driver-operator
 
-go 1.14
+go 1.15
```

Comment:
was this intentional?

Reply:
Yes, that's what ART is using: https://github.com/openshift/aws-ebs-csi-driver-operator/blob/master/Dockerfile.rhel7#L1